Tools/AP_Periph/can.cpp

```diff
@@ -544,10 +544,18 @@ void AP_Periph_FW::handle_arming_status(CanardInstance* canard_instance, CanardR
 
 
 #if defined(AP_PERIPH_HAVE_LED_WITHOUT_NOTIFY) || defined(HAL_PERIPH_ENABLE_NOTIFY)
-void AP_Periph_FW::set_rgb_led(uint8_t red, uint8_t green, uint8_t blue)
-{
+    void AP_Periph_FW::set_rgb_led(uint8_t red, uint8_t green, uint8_t blue){
```

Comment:
whitespace:
Your void is indented in and the { needs to be on the next line.
Take a look at https://ardupilot.org/dev/docs/style-guide.html . Good rule of thumb is to just follow the pattern of nearby code

Reply:
noted. astyle vscode extension installed.